java/codebuild/project/README.md

```diff
@@ -1,10 +1,10 @@
 # Welcome to your CDK Java project!
```

Comment:
This is much better, but we are trying to increase the quality of our examples READMEs so that there is little context outside of that README required to gain value from the example.

The [Typescript ALB example](https://github.com/aws-samples/aws-cdk-examples/tree/master/typescript/application-load-balancer) and [Java Stepfunctions Poller](https://github.com/aws-samples/aws-cdk-examples/tree/master/java/stepfunctions-job-poller) are great representations of complete READMEs.

Reply:
Updated